src/webui/www/public/scripts/login.js

```diff
@@ -62,3 +116,14 @@ function submitLoginForm(event) {
     // clear the field
     passwordElement.value = '';
 }
+
+document.addEventListener('DOMContentLoaded', () => {
+    const loginForm = document.getElementById('loginform');
+    loginForm.setAttribute('method', 'POST');
+    loginForm.addEventListener('submit', submitLoginForm);
+
+    document.getElementById('username').focus();
+    document.getElementById('username').select();
```

Comment:
Maybe instead set the [autofocus](https://www.w3schools.com/tags/att_input_autofocus.asp) attribute on the `username` input?

Reply:
Done.